### PR TITLE
fix(master data&publication): improve failed tc

### PR DESCRIPTION
**Pull request by fauziMK**
### Related
https://app.clickup.com/t/860rw2j1a

### Overview
Improvement tc (test case) failed status

### Changes
- fix delete data scenario master data + add assertion delete data
- fix detail scenario master data
- fix data test image in master data publication
- add assertion detail data after update process
- define selectors button save change, in form add master data service
- add function cy.intercept for hide response xhr network

### Evidence
project: Portal Jabar
title: Master data service & publication - Fix test plan timeout & failed
participants: @fauziMK @wulanLastia @farrasnaufalwork
lampiran: https://i.ibb.co/v4Q2bXB/Screenshot-2023-09-13-at-16-13-06.png

### CODE REVIEW

File: cypress/support/pages/service/service_publication/detail.cy.js [hunk 1]
```diff
@@ -132,6 +132,39 @@ export class DetailPublicationPage {
         })
     }
 
+    assertionAdditionalInformationUpdate() {
+        const newsKeyword = "cypress/fixtures/agenda/agenda_data.json"
+
+        cy.readFile(newsKeyword).then((object) => {
+            const newsInformationTable = cy.xpath('/html[1]/body[1]/div[1]/div[1]/div[1]/div[1]/main[1]/section[2]/div[1]/div[1]')
```
Comment (farrasnaufalwork) [inline cypress/support/pages/service/service_publication/detail.cy.js:139]: kang punten untuk di fungsi constanta ini apakah memang belum dibuat POM elementnya kang?

File: cypress/support/selectors/service/service_list/deletes.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    // modalsConfirmationDelete: '/html[1]/body[1]/div[1]/div[2]/div[1]/section[1]',
+    modalsConfirmationDelete: '[data-cy="modal-container"]',
+    btnYesDelete: '/html[1]/body[1]/div[1]/div[2]/div[1]/section[1]/div[3]/div[1]/button[2]',
+    btnCancelDelete: '[data-cy="agenda__modal-button--cancel"]',
+    btnUnderstand: "Saya Mengerti"
```
Comment (farrasnaufalwork) [inline cypress/support/selectors/service/service_list/deletes.js:6]: kang punten izin bertanya kembali, apakah pemanggilan selector element 'Saya Mengerti' di 'btnUnderstand' ini dapat dipanggil?

**Review by farrasnaufalwork — COMMENTED** [review]
kang @fauziMK punten nitip satu pertanyaan lagi

File: cypress/support/selectors/service/service_list/deletes.js (hunk 2 above)
Comment (fauziMK) [reply cypress/support/selectors/service/service_list/deletes.js:6]: Bisa, untuk pemanggilan di file pagesnya bisa menggunakan cy.contains kang

Comment (farrasnaufalwork) [reply cypress/support/selectors/service/service_list/deletes.js:6]: oke siap kang
terimakasih jawabannya kang Fauzi

**Review by farrasnaufalwork — APPROVED** [review]

**Pull request merged**